corehq/messaging/scheduling/forms.py

```diff
@@ -422,6 +426,10 @@ def get_layout_fields(self):
                 crispy.Field('sms_callback_intervals'),
                 data_bind="visible: $root.content() === '%s'" % ScheduleForm.CONTENT_SMS_CALLBACK,
             ),
+            crispy.Div(
+                crispy.Field('should_run_rule', id="should_run_rule"),
+                data_bind="value: should_run_rule",
```

Comment:
i don't think you need data_bind here since we are setting the value directly and not via Knockout model

Reply:
Yeah. I started out by thinking of binding to the KO model but changed the approach later and then I forgot to remove it from code.